colibre-zooms/config.yml

```diff
@@ -0,0 +1,174 @@
+# Example configuration file for a COLIBRE-zoom simulation
+
+
+# Location of your other .yml files that are passed to the
+# velociraptor auto plotter (these describe your scaling)
+# relation figures. Also required is the registration file
+# in the case where you have non-catalogue properties
+# used in the autoplotter.
+auto_plotter_directory: auto_plotter
+auto_plotter_registration: registration.py
+auto_plotter_global_mask: derived_quantities.low_interloper_halos
```

Comment:
@MatthieuSchaller global mask for autoplotter plots specified here

Reply:
Can you please add a comment here explaning what this global mask does?